lesson07/assignment/vue-project/src/components/Home.vue

```diff
@@ -0,0 +1,82 @@
+<template>
+  <!-- Jumbotron -->
+  <b-jumbotron
+    class="rounded-0 text-light text-center d-flex justify-content-center align-items-center"
+  >
+    <!-- bg image overlay div  -->
+    <div class="img-overlay"></div>
+    <b-container class="p-0">
+      <h1 class="display-3">{{ title }}</h1>
+      <p class="lead">{{ subtitle }}</p>
+      <b-alert
```

Comment:
Would like to see some more comments in this file. Especially with your use of Bootstrap/Vue elements like <b-alert> and your countdown